docs/config/index.md

```diff
@@ -1785,6 +1785,33 @@ export default defineConfig({
 })
 ```
 
+### onStackTrace
+
+- **Type**: `(error: Error, frame: ParsedStack) => boolean | void`
+
+Apply a filtering function to each frame of each stacktrace when handling errors.
+
+Can be useful for filtering out stacktrace frames from third-party libraries.
+
+```ts
+import type { ParsedStack } from 'vitest'
+import { defineConfig } from 'vitest/config'
+
+export default defineConfig({
+  test: {
+    onStackTrace(error: Error, { file }: ParsedStack): boolean | void {
+      // If we've encountered a ReferenceError, show the whole stack.
```

Comment:
Error object is always a plain object because we cannot transfer instance of error, so this is misleading. This should also be probably mentioned in docs